### PR TITLE
WIRE backend deployment to Kubernetes

### PR DESCRIPTION
**Pull request by davidmukiibi**
#### What does this PR do?
Deployment scripts for the WIRE backend to kubernetes and minikube
Separation of local (docker/docker compose & minikube) and remote (kubernetes) dev environments
#### Description of Task to be completed?
Deploy wire to minikube as the first step (play area) before full deployment it to kubernetes engine.
#### Where should the reviewer start?
[Deployments an Services](https://github.com/AndelaOSP/wire-api/tree/ch-create-kubernetes-dev-environment/wire-k8s)
#### How should this be manually tested?
TL:DR; Working on a makefile to automate it all
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#156997128](https://www.pivotaltracker.com/story/show/156997128)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Comment by kimobrian** [comment]
@davidmukiibi Sync with @RayNjeri when she comes back tomorrow to fix the tests
